runtime/compiler/env/VMJ9.cpp

```diff
@@ -145,6 +145,9 @@
 #define J9_FINDKNOWNCLASS_FLAG_EXISTING_ONLY 2
 #endif
 
+/* Max hot field list length allowed for a class if scavenger dynamicBreadthScanOrdering is enabled */
+#define MAX_HOT_FIELD_LIST_LENGTH 10
```

Comment:
This should be queried from the VM/GC somewhere I don't see why it is hardcoded in the JIT.